refactor (#336)
KxSystems/docs: docs/basics/internal.md

```

## `-18!x` (compress bytes)

> Returns IPC byte representation of `x` (as per [`-8!x`](#-8x-to-bytes)), applying compression given [IPC compression rules](ipc.md#compression):

Reviewer: ```suggestion
Returns the IPC byte representation of `x` according to [`-8!x`](#-8x-to-bytes), applying compression based on [IPC compression rules](ipc.md#compression):
```

returns `y` as a string or strings formatted as a float to `x` decimal places.
(Since V3.6 2018.09.26.)

> It is atomic and doesn’t take [`\P`](syscmds.md#p-precision) into account. e.g.

Reviewer: ```suggestion
It is atomic and doesn’t take [`\P`](syscmds.md#p-precision) into account. For example:
```
